src/web/src/app/space/invitation-list/component.ts

```diff
@@ -87,8 +88,8 @@ export class SpaceInvitationListComponent implements OnInit {
 			distinctUntilChanged(), // Optional: ensure distinct values before making requests
 			switchMap(value =>  this.auth.getList(10, 0, value).pipe(
 				map(res => {
-					this.hasUsers = res.list.length > 0;
-					return res.list;
+					this.hasUsers = res.length > 0;
+					return res;
 				}))
```

Comment:
Тут точно не ошибка? Раньше было return res.list;
Если не ошибка, то давай тогда как то так:
```suggestion
				tap(res => this.hasUsers = res.length > 0)
```

Reply:
Точно не ошибка. Готово.